x/ibc-hooks/ibc_module.go

```diff
@@ -44,12 +44,12 @@ func (im IBCMiddleware) OnChanOpenInit(
 		hook.OnChanOpenInitBeforeHook(ctx, order, connectionHops, portID, channelID, channelCap, counterparty, version)
 	}
 
-	err := im.App.OnChanOpenInit(ctx, order, connectionHops, portID, channelID, channelCap, counterparty, version)
+	finalVersion, err := im.App.OnChanOpenInit(ctx, order, connectionHops, portID, channelID, channelCap, counterparty, version)
```

Comment:
Not sure what this final version is, but ACK that its wired right

Reply:
`OnChanOpenInit` returns the version. I'm also not sure, but I think that is the modified version. I'm passing both to the After method in case it needs to act on either. We could also ignore it and if someone needs it they can use Override... just thought it was weird not to pass it down since we're passing err.